lib/modules/manager/maven-wrapper/artifacts.ts

```diff
@@ -157,6 +163,37 @@ async function executeWrapperCommand(
   }
 }
 
+function getExtraEnvOptions(
+  deps: PackageDependency<Record<string, unknown>>[]
```

Comment:
```suggestion
  deps: PackageDependency[]
```
remove default type parameter